### PR TITLE
Use table instead of figure

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -677,14 +677,25 @@ \subsubsection{Experiment}
...
+            900 & 32.02 &  9.35  \\ 
+            1300 & 36.67 & 9.41 \\ 
+            1700 & 40.55 & 9.54 \\
+            \hline
+        \end{tabular}
+    }
+\end{table}
 
-It the Fig.~\ref{fig:mam_vs_e2ee}, a comparison of the two result reveals elapsing time of E2EE is obviously less than elapsing time of MAM which shows the E2EE process presented in the paper is a much more appropriate solution for low-level embedded system to send authenticated streaming data. Next, regarding the trend of the two lines, the performance of E2EE is obviously more steady than MAM in the given range.
+It the Table~\ref{tab:mam_vs_e2ee}, a comparison of the two result reveals elapsing time of E2EE is obviously less than elapsing time of MAM which shows the E2EE process presented in the paper is a much more appropriate solution for low-level embedded system to send authenticated streaming data. Next, regarding the trend of the performance of MAM and E2EE, respectively, the performance of E2EE is obviously rising slowly than MAM in the given range.
```
Comment (jserv) [inline paper.tex]: Can you discuss the slope of a curve?

Comment (howjmay) [reply paper.tex]: Let me provide regression equation here

Comment (howjmay) [reply paper.tex]: Or I should said "the time mam cost rising faster than E2EE"?

Comment (jserv) [reply paper.tex]: Don't say "faster" in formal expressions. Always use numbers and implications.

Comment (howjmay) [reply paper.tex]: 那我算出 regression equation 這樣就可以很數值化計算了

Comment (jserv) [reply paper.tex]: Go ahead.

File: paper.tex [hunk 2]
```diff
@@ -677,25 +677,47 @@ \subsubsection{Experiment}
...
+            900 & 32.02 &  9.35  \\ 
+            1300 & 36.67 & 9.41 \\ 
+            1700 & 40.55 & 9.54 \\
+            \hline
+        \end{tabular}
+    }
+\end{table}
 
-It the Fig.~\ref{fig:mam_vs_e2ee}, a comparison of the two result reveals elapsing time of E2EE is obviously less than elapsing time of MAM which shows the E2EE process presented in the paper is a much more appropriate solution for low-level embedded system to send authenticated streaming data. Next, regarding the trend of the two lines, the performance of E2EE is obviously more steady than MAM in the given range.
+It the Table~\ref{tab:mam_vs_e2ee}, a comparison of the two result reveals elapsing time of E2EE is obviously less than elapsing time of MAM which shows the E2EE process presented in the paper is a much more appropriate solution for low-level embedded system to send authenticated streaming data.
```
Comment (jserv) [inline paper.tex]: x/low-level embedded system/resource-constrained systems/

**Pull request merged**